resources/job.rb

```diff
@@ -0,0 +1,9 @@
+actions :create, :delete
+default_action :create
+
+attribute :name, name_attribute: true, regex: /^[-_+.a-zA-Z0-9]+$/
+attribute :project, kind_of: String, required: true
+attribute :config, kind_of: Hash, default: {}
+
+attribute :endpoint, kind_of: String, default: 'https://localhost'
```

Comment:
Should we try to be smarter and point to `node['rundeck_server']['rundeck-config.properties']['grails.serverURL']` or something? I know you can't do that in the resource file and it must be done manually on the provider with a `||=` or something. We could default to `https://localhost` or `http://localhost` if no attribute is available